src/Forms/Checkout.js

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import Paper from '@material-ui/core/Paper';
```

Comment:
You're inconsistent with your use of quotes. Pick on style and stick with it. Better yet, use `prettier` to have it format the code for you.